falcon_telegraf/utils.py

```diff
@@ -10,4 +10,16 @@ def merge_and_normalize_tags(*dicts: Dict[Any, Any]):
                     result[str(k)] = str(v)
                 except:
                     pass  # don't care
+    return result
+
+
+def merge_values(*dicts: Dict[Any, Any]):
```

Comment:
Isn't this the same as merge_and_normalize_tags? Can they both be replaced by merge_something()

Reply:
e.g. metric type (tag/value) passed as parameter and if clause added to determine whether to cast as string